pom.xml

```diff
@@ -168,11 +168,19 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+        <!--
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
 			<version>5.1.36</version>
-		</dependency>
+		    </dependency>
+        -->
+
+        <dependency>
```

Comment:
I do not think we need to have the oracle ojdbc dependency here, unless we use the embedded oracle database